CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   * Added `graphdb.external-url.enforce.transactions=true`
 * Removed calculation of `lb_tls_enabled` in the LB module as it is calculated in the main.tf
 * Removed `monitoring_route53_healtcheck_fqdn_url` in favor of `graphdb_external_dns`.
+* Change default EC2 instance type r6i.2xlarge
```

Comment:
These needs to be in new version 1.2.4

Reply:
Hm, why? We don't have 1.2.3... but when I think about it changing the instance type might be more appropriate for a change in the minor version...

Reply:
Oops, I thought we had 1.2.3, sorry.